.gitignore

```diff
@@ -108,4 +108,4 @@ dist
 
 # sample configuration files
 triggers.json
-settings.json
\ No newline at end of file
+settings.json
```

Comment:
This still seems to be in the PR and shouldn't be.

Reply:
I'll remove it then.